spec/ttml1.xml

```diff
@@ -5045,6 +5045,17 @@ then a presentation processor must use the closest supported value.</p>
 the computed padding and the supported padding is minimized. If there are multiple closest supported values equally distant from
 the computed value, then the value most distant from 0, i.e., the greatest padding, is used.</p>
 </note>
+<note role="elaboration">
+	<p>Padding space is applied as inset space to the part of the region's area that corresponds to the large allocation rectangle 
+	in <bibref ref="xsl11"/>. Because the extent of a TTML region defines the fixed dimensions of the large allocation rectangle, 
+	padding effectively reduces the available space for a content rectangle in the XSL-FO model. 
+	This needs to be taken into account when the region's extent is conceptually mapped to dimensions of a content rectangle. For this 
+	mapping the sum of the padding-before value and padding-after value and the sum of padding-start value and padding-end value need 
+	to be subtracted from the region's width and height (subject to the value of the region's writingMode).
+	To get the top left vertex of the content rectangle the values for padding-start and padding-before need to be added the region's 
+	origin (subject to the value of the region's writingMode).
+	</p>		
+</note>
```

Comment:
This note is overly detailed and long, and introduces conceptual dependencies on XSL-FO which we have generally avoided elsewhere (except for section 9.3.3). I am replacing this with two notes in this section (8.2.16), one that reiterates and emphasizes the previously supplied parenthetical "(or inset)", and another just after the example that elaborates the details of insetting while explicitly describing the extent of the remaining content rectangle. In addition, I am adding a note in section (8.2.7), the definition of tts:extent, that additionally notes that padding is interior to the region's extent.